roles/filetree_create/templates/current_credential_types.j2

```diff
@@ -2,14 +2,23 @@
 controller_credential_types:
 {% for credential_type in current_credential_types_asset_value %}
   - name: "{{ credential_type.name }}"
-    description: "{{ credential_type.description }}"
+    description: "{{ template_overrides_resources.credential_type[credential_type.name].description
+                     | default(template_overrides_global.credential_type.description)
```

Comment:
Makes it sense setting all the exported `credential_types` with the same (`template_overrides_global`) description?

Reply:
I guess unlikely, but you MIGHT want to (which is what this gives the option to do. For example you might want to just set the description across all cred_types saying that this is added by duplication from a primary cluster

Reply:
I would agree if the same description should be added to the exported one, but the current proposal is setting the exact same description to every credential type, what have nonsense in my opinion.

Reply:
Sometimes it would be good to set global description, because maybe someone wants to use the description as some kind of indicator.